fw-child/template/learn/query.php

```diff
@@ -76,5 +75,12 @@ class="learn-topic-grid py-7"
 				<p class="alert alert-warning"><?php _e ( 'No items found.', 'cdc' ); ?></p>
 			</div>
 		</div>
+
+		<?php
+
+				$i++;
+			}
+
+		?>
```

Comment:
This code seems to be there in error (there is no `$i` variable, and there is an extra `}`)

Reply:
This has been taken car of in the ticket regarding the Apps page.